examples/menubar/menubar-1/js/MenubarItemLinks.js

```diff
@@ -122,6 +123,13 @@ MenubarItem.prototype.handleKeydown = function (event) {
   }
 };
 
+MenubarItem.prototype.handleClick = function (event) {
+  if (this.popupMenu) {
```

Comment:
Curious why this file is necessary?

Reply:
Hi @spectranaut !

If the selected menuitem has a submenu, then `this.popupMenu` will be true.
Those menuitems are implemented with `<a>`, but we just want them to open their submenu (aka popupMenu) when they are clicked, so we need to prevent default anchor behavior on them.

The other type of menuitem in the navigation menubar really is an anchor (semantically as well as being implemented with `<a>`), so for those, we need to keep the default anchor behavior so that it will take the user to a new page (or to a fragment, or wherever) when clicked.